codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentClientDecorator.kt

```diff
@@ -140,7 +140,7 @@ class GenericFluentClient(codegenContext: CodegenContext) : FluentClientCustomiz
                     /// ##     */
                     /// ##     .middleware_fn(|r| r)
                     ///       .build();
-                    /// let config = Config::builder().build();
+                    /// let config = Config::builder().endpoint_resolver("https://www.myurl.com").build();
```

Comment:
no, but this is a great question. `endpoint_uri` is an SDKism–it seeds the `SDK::Endpoint` builtIn. Since this depends on rules, and generic services might not have rules, we don't support this directly.

this is using that fact that `EndpointResolver<T>` is impled by `&'static str` for convenience